### ISSUE TITLE
Troubles with upgrading @fundamental-ngx/core from "0.44.0" to "^0.47.0"

### ISSUE THREAD

**Issue by egorwow70** [SAP/fundamental-ngx#11209]
Hello, fundamental-ngx Team.

Our team is currently developing the application with **"@fundamental-ngx/core": "0.44.0" and "@angular/core": "^16.2.0".**

We received a requirement to implement some part of the UI with a [generic-tag](https://sap.github.io/fundamental-ngx/#/core/generic-tag) component. We looked into the documentation and understood, so that to use it we need to upgrade the version of the library to the newest. I **switched it to the "^0.47.0"** (looking into the package-lock.json the [0.47.4](https://www.npmjs.com/package/@fundamental-ngx/core/v/0.47.4) is being fetched) and found some troubles in the application:

# 1. **fd-tab-list component is deprecated**
![image](https://github.com/SAP/fundamental-ngx/assets/41961241/d88e77ff-63f2-44f5-8387-0d12bf941be8)

We are using it inside **fd-dynamic-page with angular router-outlet**. With the upgrade of the version **the bug appeared**, that **the last tab will be always selected on the initial load** no matter what kind of URL will activate the tab. I think it can possibly be because of the non-standard approach with the router outlet that we used:
```html
<fd-dynamic-page class="home-page">
	<fd-dynamic-page-header [title]="userFirstName$ | async | heyUser">
		<fd-dynamic-page-global-actions>
			<fd-toolbar
				fdType="transparent"
				[clearBorder]="true"
			>
				<fcw-old-flex-connect-button fd-toolbar-item />
			</fd-toolbar>
		</fd-dynamic-page-global-actions>
	</fd-dynamic-page-header>
	<fd-tab-list
		[defaultTab]="findHomeNavigationTabIdFromCurrentUrl()"
		(selectedTabChange)="onSelectedHomeNavigationTabChanged($event)"
	>
		<fd-tab
			*ngFor="let homeNavigationRoute of homeNavigationRoutes; index as tabIndex"
			[id]="getHomeNavigationTabIdFromIndex(tabIndex)"
			[title]="getHomeNavigationTabTitle(homeNavigationRoute)"
		/>
		<fd-dynamic-page-content
			class="home-page__content"
			fd-scrollbar
		>
			<router-outlet />
		</fd-dynamic-page-content>
	</fd-tab-list>
</fd-dynamic-page>
```
**Is it possible to use router-outlet at all with fd-tab-list?** As for the documentation I see that it provides only the declarative way of putting components inside the page content. But I think, that lazy-loading approach with router-outlet is a huge advantage and this possibility should be there, because the page content can be huge.

Also I **tried to use the [icon-tab-bar](https://sap.github.io/fundamental-ngx/#/platform/icon-tab-bar) control** from fundamental-ngx/platform, but it seems that we cannot use it inside either fd-dynamic-page or fdp-dynamic-page. Where should we place it? **There is no documentation how to use it inside the dynamic page.**

What can I see from the documentation [fd-dynamic-page](https://sap.github.io/fundamental-ngx/#/core/dynamic-page#tabbed) still uses fd-tab-list approach:
![image](https://github.com/SAP/fundamental-ngx/assets/41961241/96674fc5-1f31-4547-96c9-5846685deed1)

And [fdp-dynamic-page](https://sap.github.io/fundamental-ngx/#/platform/dynamic-page#tabbed) controls the tabs with the help of fdp-dynamic-page-content:
![image](https://github.com/SAP/fundamental-ngx/assets/41961241/0878b2db-f78c-4d6b-a861-f36262a1b031)

**But in both approaches there is nothing about the router-outlet**

# **2. A random scroll appeared**

In the [fd-date-picker](https://sap.github.io/fundamental-ngx/#/core/date-picker) popover
![image](https://github.com/SAP/fundamental-ngx/assets/41961241/3a413881-3570-49a0-ade0-21a2962c2b88)

In the [fd-dialog](https://sap.github.io/fundamental-ngx/#/core/dialog)
![image](https://github.com/SAP/fundamental-ngx/assets/41961241/a3d588c6-0d02-4445-847a-78f26cdbbfe6)

But we don't use fd-scrollbar for both of them.

**Comment by N1XUS** [comment]
Hello @egorwow70. We have a [PR](https://github.com/SAP/fundamental-ngx/pull/11239) with example of tabbed routing, which should cover your first question.

As to the other items, is there a chance you could generate a stackblitz example where the scrollbar is visible? I wasn't able to reproduce it with our docs website.

**Comment by egorwow70** [comment]
@N1XUS 

Hello, thanks for the reply. As I can see documentation changes are located in 0.48 version of the library. Can you give some insights, when 0.48 will be released? Our team is looking forward to it, so that to migrate the app to Angular 17.

As for the scroll issue and stackblitz I forked a sample from the tabbed dynamic page in fundamental-ngx/core samples, simply added a date-picker and it seems the issue is still there - https://stackblitz.com/edit/ed53zg-gykypy?file=src%2Fapp%2Fcustom-date-picker.component.html
![image](https://github.com/SAP/fundamental-ngx/assets/41961241/7aee103d-4b24-493c-a26f-7c0165e09e49)

**Comment by egorwow70** [comment]
@N1XUS 

As for dialog I updated stackblitz as well, it will be in the same project, where the date picker issue is located - https://stackblitz.com/edit/ed53zg-gykypy?file=src%2Fapp%2Fcustom-date-picker.component.ts

If you need to close the dialog, simply comment ngOnInit lifecycle inside CustomDatePickerComponent

Same issue seems to be there:
![image](https://github.com/SAP/fundamental-ngx/assets/41961241/ac65b797-b1de-4a66-a54c-77ccd99b0761)

If I set the height to '200px', the issue is still there, and I think, we don't want to set the height, it should be adjusted automatically:
![image](https://github.com/SAP/fundamental-ngx/assets/41961241/e8ee90d4-54a2-46b1-9bf0-787f4692e995)

**Comment by egorwow70** [comment]
@N1XUS 
Will those fixes be released with 0.47 or with 0.48?
